### ISSUE TITLE
Issue with version 1.17.7 of Clp

### ISSUE THREAD

**Issue by mjp98** [jump-dev/Clp.jl#140]
After installing Clp.jl on Julia 1.8.2 in a fresh environment I get the error 

```
julia> using Clp
ERROR: InitError: You have installed version 1.17.7 of Clp, which is not supported by Clp.jl. If the version change was breaking, changes will need to be made to the Julia code. Please open an issue at https://github.com/jump-dev/Clp.jl.
```

**Comment by ViralBShah** [comment]
Perhaps worth disabling that message and seeing how the tests perform. Otherwise, The current release of Clp.jl should be fixed to Clp 1.17.6, so that it doesn't pick 1.17.7.

**Comment by odow** [comment]
This is my fault: I recently updated Clp_jll, https://github.com/JuliaPackaging/Yggdrasil/pull/5746, but the versions were too permissive here: https://github.com/jump-dev/Clp.jl/pull/141
